src/vrl/selectielijst/management/commands/load_data_from_excel.py

```diff
@@ -0,0 +1,137 @@
+import re
+from dateutil.relativedelta import relativedelta
+from xlrd import open_workbook
+
+from django.core.management.base import BaseCommand
+
+from vrl.selectielijst.models import ProcesType, Resultaat
+from vrl.selectielijst.constants import ArchiefNominaties, Procestermijnen
+
+
+# python src/manage.pload_data_from_excel /home/anna/Downloads/20170704-ontwerpselectielijst_opsomming_excel_definitief_versie2_0.xls
+# curl -X POST http://127.0.0.1:8081/api/v1/resultaten -H "Content-Type: application/json" --data @/home/anna/Downloads/test_procestermijn_nihil.json
+
+def read_xls_row_to_dict(sheet):
+    keys = [sheet.cell(0, col_index).value for col_index in range(sheet.ncols)]
+
+    for row_index in range(1, sheet.nrows):
+        yield {keys[col_index]: sheet.cell(row_index, col_index).value
+               for col_index in range(sheet.ncols)}
+
+
+def check_choice(string, choice_dict):
+    if not string:
+        return ''
+    for k, v in choice_dict.items():
+        if str(v)[:25] in string:
+            return k
+    raise Exception('"{}" is not found in choices'.format(string))
+
+
+def parse_duration(dur_str):
+    dur_str = str(dur_str)
+    dur_str = dur_str.replace(',', '.').replace('  ', ' ')
+
+    if not dur_str:
+        return
+
+    if 'Direct' in dur_str or 'vernietigen' in dur_str:
+        return relativedelta(days=0)
+
+    NL_EN = {'jaar': 'years',
+             'weken': 'weeks',
+             'maanden': 'months'}
+    num, period = dur_str.split()
+    period = NL_EN[period.lower()]
+    num = float(num)
+    if num.is_integer():
+        rel_delta = relativedelta(**{period: num})
+    else:  # non-integer periods are not supported
+        i, d = divmod(num, 1)
```

Comment:
better to use fully-spelled out variables instead of single-letter abbreviations, otherwise it hurts readability a lot
```suggestion
        quotient, remainder = divmod(num, 1)
```